packages/gql-executor/RedisStream.ts

```diff
@@ -9,7 +9,7 @@ export default class RedisStream<T> implements AsyncIterableIterator<T> {
   private stream: string
   private consumerGroup: string
   // xreadgroup blocks until a response is received, so this needs its own connection
-  private redis = new Redis(REDIS_URL)
+  private redis = new Redis(Number(REDIS_PORT), REDIS_URL)
```

Comment:
-1 this doesn't fix the problem. ioredis parses the arguments
https://github.com/luin/ioredis/blob/98ebeec1acd2067e52169f53e57e989ebbf671ff/lib/redis/index.ts#L199-L200
so both versions (master and PR) result in the same redis options
```
{
  port: 6379,
  host: 'localhost',
...
}
```
Unless there is some library reading the environment variable directly?

Reply:
Interesting @Dschoordsch I did verify in tcp dump that the dns resolver isn't adding the port number in the resolution request any longer. So, this appears to have resolved the problem. I'll continue to monitor and see if the issue arises again, but thus far the change appears to have resolved the issue.

Reply:
Then it must be fixed in the meantime and we're running different versions locally and on PPMI. I'm fairly certain it has no effect on my system because I printed the redis options with both versions. Since parsing is done by node Url module, maybe the node versions are different? I'm running v16.9.1

Reply:
@adaniels-parabol  Could this be related? https://github.com/nodejs/node/issues/13708

Reply:
I don't think so. We are also running 16.9.1 on PPMI. I've checked again, still no further errors after implementing above code. Is it safe to move forward with this PR?

Reply:
Yes, it's safe to move forward with it, but unless we had a REDIS_URL without protocol, like `example.com:1234` I don't see this changing anything. I would rather like finding the real cause. Can you try logging the options directly in the redis constructor?